deb/openmediavault/var/www/openmediavault/js/omv/module/admin/system/time/Time.js

```diff
@@ -141,6 +148,14 @@ Ext.define("OMV.module.admin.system.time.Time", {
 				allowBlank: true,
 				readOnly: true,
 				value: "pool.ntp.org"
+			},{
+				xtype: "textfield",
+				name: "ntpclients",
+				fieldLabel: _("Allow clients"),
```

Comment:
IMO the label does not reflect the intention of this configuration, the current name suggests a boolean value.

I would also like to see a hint which describes the intention of this field.

Reply:
I updated this to "Allowed clients" and added some hint text.